pmm/v3/pmm3-client-autobuild.groovy

```diff
@@ -60,7 +60,7 @@ pipeline {
                         archiveArtifacts 'uploadPath'
                         stash includes: 'uploadPath', name: 'uploadPath'
                         archiveArtifacts 'shortCommit'
-                        slackSend botUser: true, channel: '#pmm-ci', color: '#0000FF', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+                        // slackSend botUser: true, channel: '#pmm-ci', color: '#0000FF', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
```

Comment:
This was redundant and too spammy, we want rather to report only the result in this pipeline.